packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -52,12 +52,14 @@ class PlotlyDivGraph extends CoreElement {
   PlotlyDivGraph(this.framesBarChart, this.frame, [bool datum = true])
       : _processDatum = datum,
         super('div') {
-    element.id = 'graph';
+    element.id = FRAME_GRAPH;
     element.style
       ..height = '100%'
       ..width = '100%';
   }
 
+  static const String FRAME_GRAPH = 'graph_frame_timeline';
+
   // Data is adding to a ploting graph every second or every 20 frames of data
   // collected.  Otherwise plotly will lag is displaying the data.
   static const int FRAME_CHUNKING = 20;
```

Comment:
for all constants in this CL change to lowerCamelCase. Sorry I missed that last time.

Reply:
Done and everywhere.

Reply:
Done.